apps/internal/base/base.go

```diff
@@ -83,14 +83,28 @@ type AcquireTokenOnBehalfOfParameters struct {
 // AuthResult contains the results of one token acquisition operation in PublicClientApplication
 // or ConfidentialClientApplication. For details see https://aka.ms/msal-net-authenticationresult
 type AuthResult struct {
-	Account        shared.Account
-	IDToken        accesstokens.IDToken
-	AccessToken    string
-	ExpiresOn      time.Time
-	GrantedScopes  []string
-	DeclinedScopes []string
+	Account            shared.Account
+	IDToken            accesstokens.IDToken
+	AccessToken        string
+	ExpiresOn          time.Time
+	GrantedScopes      []string
+	DeclinedScopes     []string
+	AuthResultMetadata AuthResultMetadata
```

Comment:
Should we just call it "Metadata"?  naming is hard :)

Reply:
👍 I think "AuthResultMetadata" is a reasonable name for the type ("Metadata" could also work depending on what else is in the package), however the field name should be "Metadata"

Reply:
Updated